ts/features/itwallet/machine/eid/selectors.ts

```diff
@@ -34,3 +34,10 @@ export const selectCieAuthUrlOption = (snapshot: MachineSnapshot) =>
 
 export const selectIsLoading = (snapshot: MachineSnapshot) =>
   snapshot.hasTag(ItwTags.Loading);
+
+export const selectIsCieIdEidRequest = (snapshot: MachineSnapshot) =>
+  snapshot.context.identification?.mode === "cieId" &&
+  snapshot.matches({ Issuance: "RequestingEid" });
+
+export const selectIsDisplayingPreview = (snapshot: MachineSnapshot) =>
+  snapshot.matches({ Issuance: "DisplayingPreview" });
```

Comment:
Can we make clear why we need this selector instead of the `isLoading` tag? Just a few words with a comment should be enough. Thank you!

Reply:
I added a clarification comment in fcf6204c7c44e093a59afcab835de19226240af1.